### ISSUE TITLE
Release 0.12.0 ?

### ISSUE THREAD

**Issue by jrfnl** [WordPress-Coding-Standards/WordPress-Coding-Standards#962]
Looking at the [milestone](https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards/milestone/12), I was wondering if we should start preparing for the 0.12.0 release.

I'd personally would like to still see #826, #858, #897, #855 merged before then.
~Merging #950 would be a nice extra~ (Done).

@grappler Would you have a chance to update #826 over the next two weeks ? Maybe at the WCEU contributors day ? :heart_eyes: 
~~@david-binda Would you have a chance to rebase #858 over the next two weeks and to fix the last minor issue per my last remark ?~~ Superseded by #1010 & merged.
~~@barryceelen Would you have a chance to update #897 over the next two weeks based on the remarks in the PR ?~~ Done via #966 & merged.
~~@evilebottnawi Would you have a chance to update #855 over the next two weeks based on the remarks in the PR ?~~ Updated & merged.

**Comment by grappler** [comment]
Yes, I can have a look to update it.

**Comment by barryceelen** [comment]
@jrfnl A bit tied up timewise not sure I'll be able to :-(

**Comment by jrfnl** [comment]
@grappler Thanks, looking forward to it

@barryceelen Thanks for your response, I'll sort it out in that case.

**Comment by jrfnl** [comment]
Pinging @grappler @david-binda and @evilebottnawi for a status update.

**Comment by grappler** [comment]
@jrfnl The PR has been updated. Just in time... 😄

**Comment by jrfnl** [comment]
@grappler I'll try and have a proper look at this over the weekend.

**Comment by jrfnl** [comment]
All: FYI - I've made a preliminary changelog for the 0.12.0 release.

I think that aside from the above mentioned PRs, it would be good to try and fix a few more issues related to the core CS effort before dropping the release. All the same, this release is pretty feature rich by now, so IMHO we shouldn't wait that much longer.

Agreed ?

**Comment by JDGrimes** [comment]
👍

**Comment by jrfnl** [comment]
As we haven't received any response from @david-binda nor @evilebottnawi and those PRs only need a few little tweaks to make them mergeable, does anyone have any objections to me forking the PR branches, updating the code and submitting it as a new PR ?

**Comment by ntwb** [comment]
No objections here 👍 

That said you should be able to edit the existing PRs without needing to fork the PR branches:

https://help.github.com/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork/

**Comment by jrfnl** [comment]
> That said you should be able to edit the existing PRs without needing to fork the PR branches

Good point! Though only if that setting is enabled by the original PR creator and I can't actually see whether it is for either of these.

**Comment by ntwb** [comment]
I can't tell either, that said it looks like it worked, so win win 😏

**Comment by jrfnl** [comment]
All: with #826 about to be merged, I would like to propose we release version 0.12.0 this week.

As soon as #826 is merged I'll open the PR for the changelog and the release.

The only other thing I'm thinking we may or may not want to still add to this release would be the `ClassInstantiation` sniff for #919. I've got a sniff ready for this, but it's not covered by the handbook at this moment.
Shall I just pull it to `Extra` for now ? We can always moved it to `Core` later once the handbook has been adjusted (if it will be).

I propose to make 0.13.0 a release with just a singular purpose: cross-version compat for PHPCS 2.x and 3.x. I should be able to pull that soon after the release of 0.12.0.

Opinions ?

**Comment by JDGrimes** [comment]
👍 — I am fine with waiting on #919 though.

**Comment by jrfnl** [comment]
@JDGrimes Let me just pull it to `Extra` in that case. See #1033
